envoy/stream_info/stream_info.h

```diff
@@ -580,6 +580,26 @@ class StreamInfo {
    */
   virtual uint64_t bytesReceived() const PURE;
 
+  /**
+   * @param bytes_retransmitted denotes number of bytes to add to total retransmitted bytes.
+   */
+  virtual void addBytesRetransmitted(uint64_t bytes_retransmitted) PURE;
+
+  /**
+   * @return the number of body bytes retransmitted by the stream.
```

Comment:
nothing else in here calls out it's body bytes.  do we not track header bytes retransmitted? if not thoughts on naming?

Reply:
If you're planning to implement this for HTTP/1 and 2 I assume they're going to track all bytes, in which case I'd be inclined to keep the naming the same and comment that for HTTP/3 it only tracks body bytes?
actually what's the plan for H2 given things are intermingled on the same TCP connection?
cc @RyanTheOptimist